packages/core/src/percy.js

```diff
@@ -423,12 +432,89 @@ export class Percy {
     return syncMode;
   }
 
+  // This specific error will be hard coded
+  async checkForNoSnapshotCommandError() {
+    let isPercyStarted = false;
+    let containsSnapshotTaken = false;
+    logger.query((item) => {
+      isPercyStarted ||= item?.message?.includes('Percy has started');
+      containsSnapshotTaken ||= item?.message?.includes('Snapshot taken');
+
+      // This case happens when you directly upload it using cli-upload
+      containsSnapshotTaken ||= item?.message?.includes('Snapshot uploaded');
+      return item;
+    });
+
+    if (isPercyStarted && !containsSnapshotTaken) {
+      // This is the case for No snapshot command called
+      this.#displaySuggestionLogs([{
+        failure_reason: 'Snapshot command was not called',
+        reason_message: 'Snapshot Command was not called. please check your CI for errors',
+        suggestion: 'Try using percy snapshot command to take snapshots',
+        reference_doc_link: ['https://www.browserstack.com/docs/percy/take-percy-snapshots/']
+      }]);
+    }
+  }
+
+  #displaySuggestionLogs(suggestions, options = {}) {
+    if (!suggestions?.length) return;
+
+    suggestions.forEach(item => {
+      const failure = item?.failure_reason;
+      const failureReason = item?.reason_message;
+      const suggestion = item?.suggestion;
+      const referenceDocLinks = item?.reference_doc_link;
+
+      if (options?.snapshotLevel) {
+        this.log.warn(`Detected erorr for Snapshot: ${options?.snapshotName}`);
+      } else {
+        this.log.warn('Detected error for percy build');
+      }
+
+      this.log.warn(`Failure: ${failure}`);
+      this.log.warn(`Failure Reason: ${failureReason}`);
+      this.log.warn(`Suggestion: ${suggestion}`);
+
+      if (referenceDocLinks?.length > 0) {
+        this.log.warn('Refer to the below Doc Links for the same');
+
+        referenceDocLinks?.forEach(_docLink => {
+          this.log.warn(`* ${_docLink}`);
+        });
+      }
+    });
+  }
+
+  #proxyEnabled() {
+    return !!(process.env.HTTP_PROXY || process.env.HTTPS_PROXY);
+  }
+
+  async suggestionsForFix(errors, options = {}) {
+    try {
+      const suggestionResponse = await this.client.getErrorAnalysis(errors);
+      this.#displaySuggestionLogs(suggestionResponse, options);
+    } catch (e) {
+      // Common error code for Proxy issues
+      const PROXY_CODES = ['ECONNREFUSED', 'ECONNRESET', 'EHOSTUNREACH'];
+      if (!!e.code && PROXY_CODES.includes(e.code)) {
+        // This can be due to proxy issue
+        this.log.error('percy.io might not be reachable, check network connection, proxy and ensure that percy.io is whitelisted.');
+        if (!this.#proxyEnabled()) {
+          this.log.error('If inside a proxied envirnment, please configure the following environment variables: HTTPS_PROXY, HTTP_PROXY. Refer to our documentation for more details');
```

Comment:
```suggestion
          this.log.error('If inside a proxied envirnment, please configure the following environment variables: HTTP_PROXY, [ and optionally HTTPS_PROXY if you need it ]. Refer to our documentation for more details');
```
Most of the times people dont need second - and setting it would cause problems